app/models/protocol.rb

```diff
@@ -420,6 +420,13 @@ def archive(user)
     # Update all module protocols that had
     # parent set to this protocol
     if result
+      Protocol.where(parent: self).find_each do |p|
```

Comment:
Why you do this?

Reply:
this was done on develop branch but then we changed target branch to protocol_versioning feature, need to update to match it's updated logic